### PR TITLE
Fix/no label input dropdown

### PR DESCRIPTION
**Pull request by felix-ico**
this would fix https://github.com/telekom/scale/issues/827 ...not sure if it's a good idea to allow inputs without labels though

### CODE REVIEW

**Review by acstll — CHANGES_REQUESTED** [review]
Removing the label is definitely a bad idea as you suggest, for accessibility reasons. But I think we can provide allowing to hide the label, as requested, if we follow the recommendations here https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text
So we try and make sure there is always a label, either visible or not.

I can think of two options, but probably there are many more:
- We make the `label` required, and add a `hideLabelVisually` prop to make it visually hidden, and adjust the layout for the input text to be vertically aligned as in this PR ("approach 1" in the article)
- We add an `innerAriaLabel` prop, and adjust the layout when no label is present ("approach 2" in the article)

@felix-ico what do you think?

**Comment by felix-ico** [comment]
@acstll I'd go for approach #1, I'll make the necessary changes! 🙌

**Review by acstll — CHANGES_REQUESTED** [review]
Looks good! — I think we should also add the new `hideLabelVisually` prop to Storybook. Thank you!

File: packages/components/src/components/text-field/text-field.tsx [hunk 1]
```diff
@@ -241,16 +242,15 @@ export class TextField {
       'datetime-local',
     ];
 
+    console.log('HIDE LABEL ? ', this.hideLabelVisually, this.label);
```
[the remark this answers is not in the document]
Comment (felix-ico) [reply packages/components/src/components/text-field/text-field.tsx]: ?? I'm not sure what happened here but this was removed a while ago https://github.com/telekom/scale/pull/1619/commits/129a2b607c3ff30a4a81c4e5b4a655271fb20dc1

**Pull request merged**